pkg/auth/peer.go

```diff
@@ -110,6 +110,10 @@ func New(ctx context.Context, log *zap.Logger, config Config, configDir string)
 		return nil, errs.New("allowed satellites parameter '--allowed-satellites' resolved to zero satellites")
 	}
 
+	for satelliteID := range allowedSats {
+		log.Debug("allowed satellite: " + satelliteID.String())
+	}
```

Comment:
nit: it might be a good idea to make this a separate change.

Reply:
Do you want this as a separate change?

Reply:
No; it's fine